code/modules/surgery/organs/external/_external_organs.dm

```diff
@@ -144,6 +144,11 @@
 	var/mutable_appearance/appearance = mutable_appearance(sprite_datum.icon, finished_icon_state, layer = -image_layer)
 	appearance.dir = image_dir
 
+	///Also give the icon to the obj
+	if(use_mob_sprite_as_obj_sprite)
+		icon_state = finished_icon_state
+		icon = sprite_datum.icon
```

Comment:
wont this give it directional sprites? stuff like moth wings should always be facing north probably

Reply:
Oops, fixed